docs/infrastructure_and_maintenance/performance.md

```diff
@@ -52,6 +52,17 @@ In production setups:
 - Always enable opcache for php-fpm/`mod_php`.
 - Prefer php-fpm and web server using it over fast-cgi for lower overall memory usage.
 
+!!! caution
+
+    To avoid deprecations when using PHP 8.2 or 8.3, [[= product_name =]] v4.6 needs an additional error handler.
+    The error handler is present by default for projects that started with 4.6.8 or later.
+    For projects that started with earlier versions, you need to add the error handler manually by running the following commands:
+
+    ``` bash
+    composer config extra.runtime.error_handler "\\Ibexa\\Contracts\\Core\\MVC\\Symfony\\ErrorHandler\\Php82HideDeprecationsErrorHandler"
+    composer dump-autoload
+    ```
+
```

Comment:
It's clear that ["update_from_4.6"](https://ez-systems-developer-documentation--2841.com.readthedocs.build/en/2841/update_and_migration/from_4.6/update_from_4.6/) is only about 4.6 and in the continuation ["update_to_5.0" removes it](https://ez-systems-developer-documentation--2841.com.readthedocs.build/en/2841/update_and_migration/from_4.6/update_to_5.0/#update-ibexa-dxp-application).

But for "performance" when on 5.0 branch of our doc, this is less clear. It's not since 4.6 that this is needed, it's for 4.6 only. I would add something about 5.0 not needing this or I wouldn't have it on 5.0 branch but onyl on 4.6 branch.